frontend/src/components/SearchPage/CourseItem.js

```diff
@@ -141,54 +143,56 @@ const PlusWrapper = styled.span`
   color: rgb(0,97,0);
 `;
 
-const PlusIndicator = ({ value }) => {
-  return (
-    <PlusWrapper>
-      <FA name="arrow-up" style={{ marginRight: '5px' }} />
-      {value}
-    </PlusWrapper>
-  );
-}
+const PlusIndicator = ({ value }) => (
+  <PlusWrapper>
+    <FA name="arrow-up" style={{ marginRight: '5px' }} />
+    {value}
+  </PlusWrapper>
+);
 
 const MinusWrapper = styled.span`
   margin-left: 5px;
   color: rgb(156,0,6);
 `;
 
-const MinusIndicator = ({ value }) => {
-  return (
-    <MinusWrapper>
-      <FA name="arrow-down" style={{ marginRight: '5px' }} />
-      {Math.abs(value)}
-    </MinusWrapper>
-  );
-}
+const MinusIndicator = ({ value }) => (
+  <MinusWrapper>
+    <FA name="arrow-down" style={{ marginRight: '5px' }} />
+    {Math.abs(value)}
+  </MinusWrapper>
+);
 
-const getDifferenceIndicator = difToAverage => {
-  if (Math.abs(difToAverage) < .3) {
+const getDifferenceIndicator = (difToAverage) => {
+  if (Math.abs(difToAverage) < 0.3) {
```

Comment:
Is this one an ESLint thing or do you just think it's more clear? (I do think it's more clear, just wondering)

Reply:
Lint